ports/cairo/vcpkg.json

```
@@ -1,10 +1,10 @@
 {
   "name": "cairo",
   "version": "1.17.8",
-  "port-version": 2,
+  "port-version": 4,
```

```suggestion
  "port-version": 3,
```
Changes submitted by a PR need only be incremented once.

---

Yes, I just saw that in the pipeline. Sorry.